src/components/task.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Item from './item';
+
+const Task = props => (
+  <li className="task__item">
+    {props.variety !== '' ?
```

Comment:
I'd be tempted to replace `props.variety !== '' ? ...` with `props.variety ? ...` for brevity, if you can. For non-empty strings the behaviour will be the same, only differences would be that falsy values wouldn't generate a span, but I suspect that's okay?